test/ExecutionFramework.h

```diff
@@ -176,6 +173,7 @@ class ExecutionFramework
 		return encode(u256((value.numerator() << fractionalBits) / value.denominator()));
 	}
 	static bytes encode(util::h256 const& _value) { return _value.asBytes(); }
+	static bytes encode(util::h160 const& _value) { return encode(util::h256(_value, util::h256::AlignRight)); }
```

Comment:
Could also use this (to use the `encode(bytes)` below for padding:
```suggestion
	static bytes encode(util::h160 const& _value) { return encode(_value.asBytes, true); }
```